source/opt/inline_pass.cpp

```diff
@@ -0,0 +1,432 @@
+// Copyright (c) 2016 The Khronos Group Inc.
+// Copyright (c) 2016 Valve Corporation
+// Copyright (c) 2016 LunarG Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "inline_pass.h"
+#include "iterator.h"
+
+#include <unordered_map>
+
+static const int SPV_ENTRY_POINT_FUNCTION_ID = 1;
```

Comment:
I added a comment and have adopted the Google style here. I did keep the Spv prefix; it gives a little context at the use points. I also left off the Index suffix since I added the comment and the role of an index is fairly clear at the use points.